### DISCUSSION TITLE
Maps from local machine to display with GeoMap

### DISCUSSION THREAD

**Discussion by YannBinotArtal** [EclipseNebula/nebula#505] [Q&A]
Hi,

Currently, I can display the maps from OpenStreetMap server with Nebula Geomap.

But if there is not web connection, is it possible to display maps stored in my computer with Nebula Geomap ?

Thank you

**Comment by wimjongman** [comment]
If you look at the source code, you see that GeoMap supports two tile servers out of the box. In order for your software to work offline, you need to implement your own tile server. 

Here is page describing this: https://switch2osm.org/serving-tiles/manually-building-a-tile-server-20-04-lts/

**Reply by YannBinotArtal** [reply]
Thank you for your help. I will look into it immediately
